src/community/design-system-working-group/index.md.njk

```diff
@@ -14,6 +14,15 @@ The working group:
 - makes recommendations to help contributors improve their work
 - advocates for the needs of colleagues across government
 
+The working group reviews:
+
+- new patterns and components
+- changes to components that will easily be recognised
+- changes to components that will affect when they should be used
+- content that will affect how users make services - and how services are assessed
```

Comment:
Can we say who assesses services and make the last clause active?

Reply:
```suggestion
- content that will affect how users make services that meet the Service Standard
```
Slightly different meaning, but what do you think @EoinShaughnessy?

Reply:
@calvin-lau-sig7 That sounds really good. When you put it like that, it feels like the previous wording was splitting out things unnecessarily.

Reply:
Cheers @EoinShaughnessy. Yes looks a bit obvious now doesn't it?

Reply:
@calvin-lau-sig7 It always does after someone else has done the hard work 😀